src/runtime_src/core/common/sensor.cpp

```diff
@@ -210,19 +210,19 @@ read_data_driven_electrical(const std::vector<xq::sdm_sensor_info::data_type>& c
     sensor_array.push_back({"", pt});
   }
 
-  uint64_t bd_power;
+  std::string bd_power = "N/A";
   // iterate over power data, store to ptree by converting to watts.
   for (const auto& tmp : power) {
     if (boost::iequals(tmp.label, "Total Power"))
-      bd_power = tmp.input;
+      bd_power = std::to_string(tmp.input);
```

Comment:
why is tmp.input not being converted to proper unit using "unit modifier" as being done for voltage and current?

Reply:
Good point. We should do that.